components/ingest-service/backend/chef_client_run.go

```diff
@@ -31,6 +31,7 @@ type ChefClientRun struct {
 	EndTime              time.Time            `json:"end_time"`
 	Source               string               `json:"source"`
 	Status               string               `json:"status"`
+	ChefRunStatus        string               `json:"chef_run_status"`
```

Comment:
can you add a comment here explaining why this exists, for our future selves? something like the title of your pr + a couple words why?

Reply:
Turns out we didn't need this member of this struct anyway. But I added a comment to the other one.